build.sbt

```diff
@@ -10,7 +10,7 @@ libraryDependencies ++= Seq(
   ws,
   specs2 % Test,
   "com.gu" %% "pan-domain-auth-play_2-4-0" % "0.2.7",
-  "com.amazonaws" % "aws-java-sdk" % "1.7.5"
+  "com.amazonaws" % "aws-java-sdk" % "1.10.50"
```

Comment:
I've just changed the dependencies of Prism to only depend on a subset and the resulting artifact ended up reducing from 71Mb to 50Mb ~30% smaller. As a result of this experiment I would recommend this - although use a single version variable.

`core` will automatically get pulled in with either of the other two.